### PR TITLE
Updating output messages to be printed in default mode

### PR DESCRIPTION
**Pull request by saurabsa**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md).

### CODE REVIEW

File: src/dev-spaces-preview/azext_dev_spaces_preview/custom.py [hunk 1]
```diff
@@ -119,7 +119,11 @@ def _install_dev_spaces_cli(force_install):
 
     if should_install_azds:
         # Install AZDS
-        logger.info('Installing Dev Spaces (Preview) commands...')
+        print('Installing Dev Spaces (Preview) commands...')
+        if system == 'Windows':
+            print('A separate window will open to guide you through the installation process.')
+        else:
+            print('The following dependencies will be installed: curl, unzip, dotnet.')
```
Comment (williexu) [inline src/dev-spaces-preview/azext_dev_spaces_preview/custom.py]: Please use the logger instead of the print() method

Comment (saurabsa) [reply src/dev-spaces-preview/azext_dev_spaces_preview/custom.py]: Yes, that's what. logger.info outputs something when we have the --verbose flag. What if we want to output something by default (even w/o verbose)?

Comment (rakeshvanga) [reply src/dev-spaces-preview/azext_dev_spaces_preview/custom.py]: Also for the dependencies message, I have already added that line to linux install script. So no need to output that message here.

Comment (williexu) [reply src/dev-spaces-preview/azext_dev_spaces_preview/custom.py]: In the CLI, we try not to make commands act like scripts, where it is common for developers to output all information to the user and update them on progress.

If you really want to send a message during command execution, use `logger.warning()`.

Comment (saurabsa) [reply src/dev-spaces-preview/azext_dev_spaces_preview/custom.py]: logger.warning() help then

**Pull request merged**